phonebook-ui/src/app/auth/forgot-password-page/forgot-password-page.component.css

```diff
@@ -0,0 +1,151 @@
+.auth-form {
```

Comment:
what is this? Is it source or result? In git there should be only sources and instruction, if needed, how to transpile them into end files